### PR TITLE
good practices to attach events with jQuery

### PR DESCRIPTION
**Pull request by pedrohenriquerls**
The idea is avoid chain the .off and .on, when you will attach some event anywhere.
### Why not do it
The jQuery use a events queue to know what is the callback he will call, when you use off first(to avoid assign the same event more than one time), this event go to end of queue, so imagine this scenario...
```html
<div class="blastoise">
  <button class="ok-btn"></button>
</div>
```
```javascript
// Bad case: if you click on ok-btn, the parent will respond the event first
$('.ok-btn').off().on('click', function() {console.log('click! click!!')});
$('.blastoise').on('click', function() {console.log('parent!!')});

// Good: In this case the ok-btn will execute exactly the right callback and avoid any other callback
$('.ok-btn').on('click', function(e) {
  e.stopPropagation();
  console.log('click! click!!');
})
$('.blastoise').on('click', function(){console.log('parent!!')});
```

### CODE REVIEW

**Comment by lucasalencar** [comment]
For me its awesome! :shipit:

**Comment by guih** [comment]
@pedrohenriquerls as I said in slack I think you need to put the PR description's explanation in the changes it self so future reader can understand why them should follow this style.

After this I think it's good to be merged!

@pcasaretto and @cnascimentord what do you think about this?

File: README.md [hunk 1]
```diff
@@ -1968,6 +1968,22 @@ Other Style Guides
     $sidebar.find('ul').hide();
     ```
 
+  <a name="jquery--attach-events"></a><a name="25.5"></a>
+  - [25.5](#jquery--attach-events) Avoid mess up with the events queue.
```
Comment (pcasaretto) [inline README.md]: messing up

**Comment by pcasaretto** [comment]
LGTM! :shipit:

**Comment by pedrohenriquerls** [comment]
@guih done :)

**Comment by guih** [comment]
Manda ver @pedrohenriquerls :shipit:

**Pull request merged**